cmake/GridToolsConfig.cmake.in

```diff
@@ -17,7 +17,9 @@ if (@OPENMP_AVAILABLE@)
     find_dependency(OpenMP COMPONENTS CXX)
 endif()
 
-include("${GRIDTOOLS_CONFIG_CMAKE_DIR}/GridToolsTargets.cmake")
+if( NOT @PROJECT_NAME@_TARGETS_EXPORTED )
```

Comment:
What's the scenario you need to protect here?

Reply:
Say you have case
```
project(atlas-bundle)
    add_subdirectory( gridtools )
    add_subdirectory( atlas )
```
```
project(atlas)
    find_package( gridtools )
```
Because gridtools has already defined the actual targets in `add_subdirectory( gridtools )`, it is illegal to include the file `GridToolsTargets.cmake` that will create the "interface targets", which would happen within `find_package( gridtools )`

If you don't want to rely on the variable `@PROJECT_NAME@_TARGETS_EXPORTED` you could also choose instead:
```
if( NOT @PROJECT_NAME@_BINARY_DIR AND NOT TARGET  <<<INSERT A KNOWN GRIDTOOLS TARGET HERE>>> )
```